Lib/gftools/packager.py

```diff
@@ -1772,7 +1841,7 @@ def _git_fetch_master(repo: pygit2.Repository, remote_name: str) -> None:
   # using just 'master' instead of 'refs/heads/master' works as well
   stats = remote.fetch(['refs/heads/master'], callbacks=PYGit2RemoteCallbacks())
```

Comment:
This doesn't work on OS X. We get the same ssh issue which your last commit fixed. I just commented out this line and replace it with

```
subprocess.run(['git', 'fetch', 'upstream'],
                             cwd="/Users/marcfoley/Type/fonts",
                             check=True,
                             stdout=subprocess.PIPE) 
```

and it worked, https://github.com/google/fonts/pull/2664.

Reply:
@m4rc1e it's done in cf7bcd4. Maybe you can give it another test run.

Reply:
Sure thing.